src/shared/settings.d.ts

```diff
@@ -6,7 +6,8 @@
 
 import type { Rectangle } from "electron";
 
-export interface Settings {
+export interface SettingsV1 {
```

Comment:
why introduce this versioning scheme? i don't like it

just add handling for the case that reading state.json fails with ENOENT and transfer the state data from settings.json